cmake/platform.cmake

```diff
@@ -3,7 +3,7 @@
 function(target_link_platform_libraries target)
 
 if (APPLE)
-    target_link_libraries(${target} "-framework Cocoa")
+    target_link_libraries(${target} "-framework Cocoa -framework CoreServices")
```

Comment:
fsevents requires CoreServices to be linked

Reply:
https://developer.apple.com/library/archive/documentation/Darwin/Conceptual/FSEvents_ProgGuide/UsingtheFSEventsFramework/UsingtheFSEventsFramework.html#//apple_ref/doc/uid/TP40005289-CH4-SW4